tests/acceptance/config/behat.yml

```diff
@@ -211,6 +211,21 @@ default:
         - OcisConfigContext:
         - PublicWebDavContext:
 
+    apiFullTextSearch:
+      paths:
+        - '%paths.base%/../features/apiFullTextSearch'
+      context: *common_ldap_suite_context
+      contexts:
+        - SettingsContext:
+        - GraphContext:
+        - SpacesContext:
+        - PublicWebDavContext:
+        - FeatureContext: *common_feature_context_params
+        - SearchContext:
+        - CapabilitiesContext:
+        - FilesVersionsContext:
+        - OCSContext:
```

Comment:
```suggestion
        - OCSContext:
        -  TrashbinContext
```

Reply:
something not ported to stable-3

Reply:
@saw-jan This PR https://github.com/owncloud/ocis/pull/6627, adding `TrashbinContext` in every suite is not needed in the master.

Reply:
we can check if we can port that to stable branch

Reply:
@grgprarup anyone going for it?

Reply:
CC @SagarGi @SwikritiT

Reply:
if its in master than we can port it to stable as well. Making it same as it is in master would be good.

Reply:
I can make a backport PR if no one has started yet

Reply:
Chances are that they might not run in stable. I'll take a look